### ISSUE TITLE
Page: Revisit the info pages of the ACL Anthology and adapt them to the IR Anthology as needed.

### ISSUE THREAD

**Issue by potthast** [ir-anthology/ir-anthology#23]
See also Issue https://github.com/ir-anthology/ir-anthology/issues/22

**Comment by najtin** [comment]
eg: faq

**Comment by potthast** [comment]
Yes, for example the FAQ. Also the contribution instructions. There's no hurry about this one, and we need to discuss the actual procedures anyway first, and whether the ones from the ACL Anthology can be adopted.
